lib/testing/src/lib/core/pages/context-menu.page.ts

```diff
@@ -0,0 +1,41 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { BrowserActions } from '../utils/browser-actions';
+import { element, by, ElementFinder } from 'protractor';
+import { BrowserVisibility } from '../public-api';
+
+export class ContextMenuPage {
+    contextMenu: ElementFinder = element(by.id('adf-context-menu-content'));
+
+    async isContextMenuDisplayed(): Promise<boolean> {
+        try {
+            await BrowserVisibility.waitUntilElementIsVisible(this.contextMenu);
+            return true;
+        } catch (error) {
+            return false;
+        }
+    }
+
+    async clickContextMenuActionNamed(actionName: string): Promise<void> {
+        await BrowserActions.click(
```

Comment:
not sure is the right indentation

Reply:
I think prettier is asking to have it like that.

Reply:
Yes, it because of Prettier, but if it is not the wanted indentation I can edit it so it could be in a single row.